AdyenDropIn/Utilities/ComponentManager.swift

```diff
@@ -50,7 +50,7 @@ internal final class ComponentManager {
 
         // Paid section
         let amountString: String = remainingAmount.map(\.formatted) ??
-            localizedString(.amount, configuration.localizationParameters)
+            localizedString(.partialPaymentPayRemainingAmount, configuration.localizationParameters)
```

Comment:
Why this change? How is it connected to a address identifier?

This also doesn't look correct.
In case there is no `remainingAmount` (that is probably going to be invalid state, but still) you will have footer "Select payment method for the remaining Select payment method for the remaining"  🤔

Reply:
There were some conflicts with develop, amount wasn't inside `LocalizationKey.swift`

Reply:
true, this is wrong, this one should be `.amount`

Reply:
Fixed